### PR TITLE
Task/gv 762 update header after change password

### PR DESCRIPTION
**Pull request by Anchello**

### CODE REVIEW

File: packages/shared/components/notifications-widget/notifications-widget.tsx [hunk 1]
```diff
@@ -5,12 +5,12 @@ import * as React from "react";
...
 import { RingIcon } from "shared/components/icon/ring-icon";
 
-interface INorificationsWidgetProps {
+interface INotificationsWidgetProps {
   openNotifications(): void;
   notificationsCount: number;
 }
 
-const NorificationsWidget: React.FC<INorificationsWidgetProps> = ({
+const NotificationsWidget: React.FC<INotificationsWidgetProps> = ({
```
Comment (pavelzubov) [inline packages/shared/components/notifications-widget/notifications-widget.tsx]: Rename to `Props`

File: packages/shared/components/notifications-widget/notifications-widget.tsx [hunk 2]
```diff
@@ -5,12 +5,12 @@ import * as React from "react";
 import Chip, { CHIP_TYPE } from "shared/components/chip/chip";
 import { RingIcon } from "shared/components/icon/ring-icon";
 
-interface INorificationsWidgetProps {
+interface INotificationsWidgetProps {
```
Comment (pavelzubov) [inline packages/shared/components/notifications-widget/notifications-widget.tsx]: Move under component

File: packages/shared/components/notifications-widget/notifications-widget.tsx (hunk 1 above)
Comment (Anchello) [reply packages/shared/components/notifications-widget/notifications-widget.tsx]: Done

File: packages/shared/components/notifications-widget/notifications-widget.tsx (hunk 2 above)
Comment (Anchello) [reply packages/shared/components/notifications-widget/notifications-widget.tsx]: Done

**Pull request merged**